Tests/SiddhiCoreTests/Resources/Extensions4/pom.xml

```diff
@@ -1,4 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c)  2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Incorrect licenses year, it should be 2018

Reply:
This file was added on 2017. That's why I added 2017 as the license year